client/apollo/css/src/Accordion/AccordionCommon.scss

```
@@ -0,0 +1,112 @@
+@use "../common/breakpoints" as breakpoints;
+
+.af-accordion {
+  --summary-areas: "icon title-container tag-container info1 info2 arrow";
+  --summary-columns: min-content max-content max-content 1fr max-content
+    max-content;
+  --title-container-areas: "title" "subtitle";
+  --title-container-columns: 1fr;
+
+  border-radius: var(--accordion-border-radius);
+  font-size: var(--accordion-font-size);
+  line-height: var(--accordion-line-height);
+  color: var(--accordion-text-color);
+  background-color: var(--accordion-bg-color);
+
+  &__summary {
+    grid-template-areas: var(--summary-areas);
+    grid-template-columns: var(--summary-columns);
+
+    .af-accordion {
+      &__icon {
+        grid-area: icon;
+      }
+
+      &__title-container {
+        display: grid;
+        grid-area: title-container;
+        grid-template: var(--title-container-areas);
+        gap: 0.125rem 1rem;
+
+        .af-accordion {
+          &__title {
+            grid-area: title;
+            font-size: var(--accordion-font-size);
+            font-weight: 600;
+            line-height: var(--accordion-line-height);
+            color: var(--accordion-title-text-color);
+          }
+
+          &__subtitle {
+            grid-area: subtitle;
+            font-size: var(--accordion-subtitle-font-size);
+            font-weight: 400;
+            line-height: var(--accordion-line-height);
+            color: var(--accordion-subtitle-text-color);
+          }
+        }
+      }
+
+      &__tag-container {
+        grid-area: tag-container;
+      }
+
+      &__date {
+        grid-area: date-container;
+      }
+
+      &__info1 {
+        grid-area: info1;
+        color: var(--accordion-info1-text-color);
+      }
+
+      &__info2 {
+        grid-area: info2;
+        color: var(--accordion-info2-text-color);
+      }
+    }
+  }
+
+  &--primary {
+    --summary-areas: "icon title-container tag-container info1 info2 arrow";
+    --summary-columns: min-content 1fr 1fr 1fr max-content max-content;
+
+    .af-accordion__summary {
+      .af-accordion {
+        &__date {
+          display: none;
+        }
+      }
+    }
+
+    @media (width < #{breakpoints.$breakpoint-md}) {
+      --summary-areas: "icon title-container arrow" ". tag-container ."
+        ". info1 ." ". info2 .";
+      --summary-columns: min-content 1fr min-content;
+    }
+  }
+
+  &--secondary {
+    --summary-areas: "tag-container date-container title-container info2 arrow";
+    --summary-columns: max-content min-content 1fr max-content max-content;
+    --title-container-columns: max-content;
+
+    .af-accordion__summary {
+      .af-accordion {
+        &__icon {
+          display: none;
+        }
+
+        &__info1 {
+          display: none;
+        }
+      }
+    }
```

Je n'ai jamais testé de faire ça mais ça peut fonctionner je pense 🤔 

```suggestion
    .af-accordion__summary .af-accordion {
      &__icon {
        display: none;
      }

      &__info1 {
        display: none;
      }
    }
```